pkg/kn/commands/types.go

```
@@ -78,14 +80,21 @@ func (params *KnParams) Initialize() {
 	}
 }
 
-func (params *KnParams) newServingClient(namespace string) (clientservingv1.KnServingClient, error) {
-	restConfig, err := params.RestConfig()
-	if err != nil {
-		return nil, err
-	}
+func (params *KnParams) newServingClient(namespace string, cmd *cobra.Command) (clientservingv1.KnServingClient, error) {
+	dir := cmd.Flag("in-dir").Value.String()
+	switch dir {
+	case "":
+		restConfig, err := params.RestConfig()
+		if err != nil {
+			return nil, err
+		}
 
-	client, _ := servingv1client.NewForConfig(restConfig)
-	return clientservingv1.NewKnServingClient(client, namespace), nil
+		client, _ := servingv1client.NewForConfig(restConfig)
```

Why are we ignoring error here?